packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js

```diff
@@ -8,7 +8,8 @@ import {
 
 export default class TrafficController {
   /* @ngInject */
-  constructor($translate, Alerter, networkSecurityService) {
+  constructor($stateParams, $translate, Alerter, networkSecurityService) {
+    this.$stateParams = $stateParams;
```

Comment:
instead of injecting `$stateParams` in the controller, you should define the given params in your routing file (resolve section) and bind them to your component

```
resolve : {
      ip: /* @ngInject */ ($state) => () => $state.params.ip,
...
}
```